src/Symfony/Component/Process/Messenger/ExecuteProcess.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Process\Messenger;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class ExecuteProcess
```

Comment:
What about `ProcessMessage` and `ProceMessageHander`?
It looks like you're going to fix all my wish list pretty fast ;)
https://speakerdeck.com/fabpot/s

Reply:
Hah, I did read that in your slide but didn't make the connection that this is what you were referring to!

Reply:
ExecuteProcessMessage ?

Reply:
At a second glance `ProcessMessage` does read like _processing a message_...

Reply:
That's why I proposed ExecuteProcessMessage, "ExecuteProcess" (what it did) and "Message" is just the suffix

Reply:
ExecuteProcessMessage and ExecuteCommandMessage, or RunProcessMessage/RunCommandMessage?

Reply:
I like the "Run" prefix 👍